behind-the-veil-siteroot/imports/ui/components/pages/home/HomePage.jsx

```diff
@@ -221,6 +176,9 @@ export const HomePage = () => {
                     </Link>
                 </form>
             </div>
+
+            {/* this is required for the search bar to not move weirdly */}
+            <div className={"h-[50px]"} />
```

Comment:
This is kinda sus but I'm not sure what a better solution would be atm haha

Reply:
HAHA yeah very sus, but i cant get it to work without it (because the suggestions are absolute positioning with "bottom") on a screen too big it pushes it up because it doesn't scroll so I need all pages to be scrolling. once again yes very dodgy